SeQuant/core/expressions/expr_operators.hpp

```diff
@@ -149,6 +150,42 @@ inline ExprPtr operator/(const ExprPtr &lhs, const Constant &rhs) {
   return lhs * ex<Constant>(1.0 / rhs.value());
 }
 
+template <typename T>
+  requires(std::constructible_from<Variable, T>)
+ExprPtr operator+(T &&lhs, const ExprPtr &rhs) {
+  return ex<Variable>(std::forward<T>(lhs)) + rhs;
+}
+
+template <typename T>
+  requires(std::constructible_from<Variable, T>)
+ExprPtr operator+(const ExprPtr &lhs, T &&rhs) {
+  return lhs + ex<Variable>(std::forward<T>(rhs));
+}
+
+template <typename T>
+  requires(std::constructible_from<Variable, T>)
+ExprPtr operator-(T &&lhs, const ExprPtr &rhs) {
+  return ex<Variable>(std::forward<T>(lhs)) - rhs;
+}
+
+template <typename T>
+  requires(std::constructible_from<Variable, T>)
+ExprPtr operator-(const ExprPtr &lhs, T &&rhs) {
+  return lhs - ex<Variable>(std::forward<T>(rhs));
+}
+
+template <typename T>
+  requires(std::constructible_from<Variable, T>)
+ExprPtr operator*(T &&lhs, const ExprPtr &rhs) {
+  return ex<Variable>(std::forward<T>(lhs)) * rhs;
+}
+
+template <typename T>
+  requires(std::constructible_from<Variable, T>)
+ExprPtr operator*(const ExprPtr &lhs, T &&rhs) {
+  return lhs * ex<Variable>(std::forward<T>(rhs));
+}
+
```

Comment:
Why are these marked as being added in this PR? They should already be in `master`, no?